src/app/stream-deck/async-module.ts

```diff
@@ -26,24 +27,18 @@ export async function sendToStreamDeck(msg: SendToStreamDeckArgs) {
   }
 }
 
-const installFarmingObserver = _.once(() => {
-  const state = store.getState();
-  // send initial state
-  sendToStreamDeck({
-    action: 'farmingMode',
-    data: Boolean(state.farming.storeId),
-  });
-  // observe farming mode state
-  observeStore(
-    (state) => state.farming.storeId,
-    (_, newState) => {
+function createFarmingObserver(): StoreObserver<string | undefined> {
+  return {
+    id: STREAM_DECK_FARMING_OBSERVER_ID,
```

Comment:
To replicate the original behavior you probably want `runInitially`. The old one actually ran `sendToStreamDeck` twice initially!